### PR TITLE
added note about serial behaviour

### PR DESCRIPTION
**Pull request by bcoca**
##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

docs
##### ANSIBLE VERSION

```
all
```

### CODE REVIEW

**Comment by abadger** [comment]
+1 from me.  Wait for review from @dharmabumstead before merging and cherry-picking

**Review by dharmabumstead — APPROVED** [review]
Approved with some edits. Thanks @bcoca

**Pull request merged**
